### ISSUE TITLE
kld_loss diference between mse and others

### ISSUE THREAD

**Issue by aamacaya** [orybkin/sigma-vae-tensorflow#2]
Hi! I have a question. 

When using mse loss, you define the kld loss as:

`kld_loss = -tf.reduce_mean(0.5 * (1 + z_log_sigma_sq - z_mu ** 2 - tf.exp(z_log_sigma_sq)))`

But when you use the other losses, u define the kld loss as:

`kld_loss = -tf.reduce_sum(0.5 * (1 + z_log_sigma_sq - z_mu ** 2 - tf.exp(z_log_sigma_sq)))`

Why do you use reduce_mean for the first case and reduce_sum for the others?

**Comment by orybkin** [comment]
Hi! The MSE loss follows the initial implementation that I took from here: https://github.com/LynnHo/VAE-Tensorflow. It uses reduce_mean. This is wrong, and I only included it for didactic purposes. You should always use reduce_sum for both MSE and KLD.

**Comment by aamacaya** [comment]
Ok, thanks!
